--- a/src/pages/CardDetails.jsx
+++ b/src/pages/CardDetails.jsx
@@ -432,7 +432,13 @@
               {" • "}
               {item.data}
               {" • "}
-              Parcelas: {item.parcelas || 1}x
+              {
+                item.totalParcelas
+                ?
+                `💳 Parcela ${item.parcelaAtual}/${item.totalParcelas}`
+                :
+                `Parcelas: ${item.parcelas || 1}x`
+              }
 
             </small>
 
