--- a/src/pages/CardDetails.jsx
+++ b/src/pages/CardDetails.jsx
@@ -74,8 +74,21 @@
 
   const fatura =
   compras.reduce(
-    (total,item)=>
-      total + Number(item.valor || 0),
+    (total,item)=>{
+
+      if(
+        item.parcelaAtual &&
+        item.parcelaAtual !== 1
+      ){
+
+        return total;
+
+      }
+
+
+      return total + Number(item.valor || 0);
+
+    },
     0
   );
 
