--- a/src/pages/CardDetails.jsx
+++ b/src/pages/CardDetails.jsx
@@ -432,7 +432,15 @@
               {" • "}
               {item.data}
               {" • "}
-              Parcelas: {item.parcelas || 1}x
+              Parcelas: {
+                item.totalParcelas
+                ||
+                item.parcelas?.length
+                ||
+                item.parcelas
+                ||
+                1
+              }x
 
             </small>
 
